src/Commands.php

```diff
@@ -11,7 +11,6 @@
 use P4\MasterTheme\Commands\SaveCloudflareKey;
 use P4\MasterTheme\Commands\FixOrphans;
 use P4\MasterTheme\Commands\GFAddonsDisconnect;
-use P4\MasterTheme\Migrations;
```

Comment:
The Migration namespace is used [here](https://github.com/greenpeace/planet4-master-theme/blob/b529a81fc1102c327f7ffa3f4e2661d034b92ad8/src/Commands.php#L38).

Reply:
Right, but it's relative to the namespace we're in which is `P4\MasterTheme`, so it resolves properly by itself as `\P4\MasterTheme\Migrations\M004UpdateMissingMediaPath`